docs/source/top_level_elements/subject.rst

```diff
@@ -0,0 +1,497 @@
+
+.. code-block:: xml
+
+    <subject>
+        <temporal>1970-09-30</temporal>
+    </subject>
+
+.. code-block:: turtle
+
+    @prefix schema: <http://schema.org/> .
+
+    <https://example.org/objects/1> schema:temporalCoverage "1970-09-30" .
+
+Geographic subjects
+-------------------
+
+UTK has geographic subjects with and without URIs. Like with other elements, the placement of the URIs is not consistent.
+URIs will be used when present, but strings can be used when there is no URI.
+
+    mods:subject[@valueURI]/mods:geographic
+    mods:subject/mods:geographic[@valueURI]
+
+`Here's an example where the URI is present on the subject - webster:1127 <https://digital.lib.utk.edu/collections/islandora/object/webster%3A1127/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <subject authority="geonames" valueURI="http://sws.geonames.org/4050810">
+        <geographic>The Sawteeth</geographic>
+        <cartographics>
+            <coordinates>35.64342, -83.36237</coordinates>
+        </cartographics>
+    </subject>
+    <subject authority="geonames" valueURI="http://sws.geonames.org/4609260">
+        <geographic>Brushy Mountain</geographic>
+        <cartographics>
+            <coordinates>35.67787, -83.43016</coordinates>
+    </cartographics>
+    </subject>
+    <subject authority="lcsh" valueURI="http://id.loc.gov/authorities/subjects/sh85057008">
+        <geographic>Great Smoky Mountains (N.C. and Tenn.)</geographic>
+    </subject>
+
+`Here's an example where the URI is present on the geographic element - roth:2165 <https://digital.lib.utk.edu/collections/islandora/object/roth%3A2165/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <subject>
+        <geographic authority="geonames" valueURI="http://sws.geonames.org/4178924/about.rdf">Yulee Sugar Mill Ruins Historic State Park</geographic>
+    </subject>
+
+Regardless of URI placement, we will map the values the same. Note that if the geographic term includes coordinates and
+a geonames URI, we will drop the coordinates. More information on this is given in the Coordinates section following this
+section. Below is the decision for webster:1127.
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1> dcterms:spatial <http://sws.geonames.org/4050810> ;
+        dcterms:spatial <http://sws.geonames.org/4609260> ;
+        dcterms:spatial <http://id.loc.gov/authorities/subjects/sh85057008> .
+
+If only strings are present, like with volvoices:14173 <https://digital.lib.utk.edu/collections/islandora/object/volvoices%3A14173/datastream/MODS/view>`_, then the string value will be kept.
+
+.. code-block:: xml
+
+    <subject>
+        <geographic>Covington (Tenn.)</geographic>
+    </subject>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1> dcterms:spatial "Covington (Tenn.)" .
+
```

Comment:
Just pointing out about spatial.  I don't care a ton, but I felt like I should at least mention. This property has a http://purl.org/dc/dcam/rangeIncludes (you can think of this as something like an rdfs:range / literally defined as "A suggested class for values of this property.") of http://purl.org/dc/terms/Location.

First, because it has a rangeIncludes it expects the value to be an instance of that class.  In this case, http://purl.org/dc/terms/Location. Obviously we have a string literal which is for sure not an instance of this class.  We've dealt with this a lot (see our use of edm) so I don't feel strongly, but in all those case we've minimally had a URI even if the URI was not an instance of what the range like property expected. A location is defined as "A spatial region or named place." We have that so perhaps we don't care.